tools/llm_bench/task/image_generation.py

```diff
@@ -14,18 +14,31 @@
 import llm_bench_utils.metrics_print as metrics_print
 import llm_bench_utils.gen_output_data as gen_output_data
 import llm_bench_utils.parse_json_data as parse_json_data
+from llm_bench_utils.config_class import TASK
+from transformers.image_utils import load_image
+import openvino as ov
+import numpy as np
 
 FW_UTILS = {'pt': llm_bench_utils.pt_utils, 'ov': llm_bench_utils.ov_utils}
 
 DEFAULT_INFERENCE_STEPS = 20
 LCM_DEFAULT_INFERENCE_STEPS = 4
 DEFAULT_IMAGE_WIDTH = 512
 DEFAULT_IMAGE_HEIGHT = 512
+DEFAULT_STRENGTH = 1.0
```

Comment:
do we need this? Maybe better let pipeline decide which guidance scale is default for it if not given? Number of actual inference steps we can get from callback if it is required

Reply:
removed